server/odc-service/src/main/java/com/oceanbase/odc/service/collaboration/project/ProjectService.java

```diff
@@ -193,6 +192,34 @@ public void createProjectIfNotExists(@NotNull User user) {
         userResourceRoleRepository.batchCreate(userResourceRoleEntities);
     }
 
+    /**
+     * Create a built-in project for migrate user
+     *
+     * @param user user
+     */
+    @SkipAuthorize("odc migrate usage")
+    @Transactional(rollbackFor = Exception.class)
+    public ProjectEntity createProject4Migrate(@NotNull User user) {
+        // default name
+        String projectName = BUILTIN_PROJECT_PREFIX + "INNER";
```

Comment:
what does 'inner' mean?
may just simplify to 'default project'?

Reply:
just for reuse this constant.
Of course.

Reply:
just for reuse this constant.
Of course.